include/tuvx/grid.hpp

```diff
@@ -1,14 +1,15 @@
-// Copyright (C) 2023-2024 National Center for Atmospheric Research
-// SPDX-License-Identifier: Apache-2.0
-//
-/// @file Grid dimensions
+/* Copyright (C) 2023-2024 National Center for Atmospheric Research
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
@mattldawson Did we discuss which comment style we preferred?  I think we were leaning towards the single line `//` as opposed to the block `/* ... */` approach?

Reply:
I think @boulderdaze put a question to the group in the #musica-development Slack channel about what people prefer and I think there was one vote for the `/* ... */` style. Personally, I would lean to the `//` style only because I associate `//` with C++ code and `/* ... */` with C code, but I don't have a strong preference one way or the other.

Reply:
Yeah, I think it comes down to a matter of taste. For muli-line comments for C++ code, some books([C++ Programming/Code/Style Conventions/Comments](https://www.google.com/search?q=wiki+books&rlz=1C5GCCM_en&oq=wiki+books+&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQABiABDIHCAIQABiABDIJCAMQABgKGIAEMgkIBBAAGAoYgAQyCQgFEAAYChiABDIGCAYQRRg8MgYIBxBFGDzSAQgxNTgwajBqN6gCALACAA&sourceid=chrome&ie=UTF-8)) recommend `/* ... */` over the single line style. I also don't have a strong opinion on the style for this kind. What do you think? I think we can still decide which style we want to apply to our code base

Reply:
I'm okay with either so long as we're consistent.  The only thing that made me lean towards `//` comments as well was that in VSCode it is one key-binding `Ctrl+/` to add or remove the single line comments.  Adding/removing block comments is more tedious with having to select just the right amount of code and having to use `Shift+Alt+A`.  This isn't impossible, just a few more steps.

Reply:
Sure, I am happy to change them to use single line style.
Hi @K20shores, @sjsprecious, we are trying to decide which style to use for code blocks. I think we are leaning towards the single line style, do you have any thoughts on this?

Reply:
Thanks @boulderdaze for working on this. I am fine with the single line style and that is what I am using as well.

Reply:
That style is fine with me